ext/pdo_mysql/pdo_mysql.stub.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+/** @generate-class-entries */
+
+/** @not-serializable */
+class PdoMysql extends PDO
```

Comment:
I'm wondering if we should use the same casing as the original class: e.g. `PDOMySql` etc.?

Reply:
see https://chat.stackoverflow.com/transcript/message/56464097#56464097
and https://chat.stackoverflow.com/transcript/message/56500210#56500210

Reply:
Exact spelling of class names is an aesthetic choice, so it's fine for everyone to have their own opinion, and normal for people to be able to conjure up arguments that just so happy to support their own gut instinct. Which I will now do.

I personally chose the one in the RFC as Tim convinced me that "ucfirst(strtolower(...)) for acronyms" as `PCGOneseq128XSLRR64` is almost as horrible as `PdOsQlItE`.

I'm also pretty sure that having the class name (not just the namespace) contain Pdo is going to make the class easier to google as `PdoMysql` will be treated as a single word by search engines, whereas `Pdo\Mysql` is treated as two words, and will be more likely to bring up all of the existing pages that talk about PDO and MySQL, rather than the new manual page for the new class.